pkgs/development/libraries/miopen/default.nix

```diff
@@ -92,6 +86,12 @@ in stdenv.mkDerivation (finalAttrs: {
     hip
     clang
     llvm
+    latex
+    doxygen
+    sphinx
+    python3Packages.sphinx_rtd_theme
+    python3Packages.breathe
+    python3Packages.myst-parser
```

Comment:
I would recommend to set strictDeps to avoid such errors in the future.

Reply:
I don't quite understand what you mean.
What errors?

Reply:
That packages end up in buildInputs which should truly be in nativeBuildInputs

Reply:
Okay. New PR puts them back in buildInputs so should be fine there.